aws-ecr-login/aws-ecr-login.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: aws-ecr-login
```

Comment:
It would be great if you could add the necessary metadata (see below) so that Tekton Hub and other supporting tools (in development) is able to search the task.

  ```yaml
  metadata:
    labels:
      app.kubernetes.io/version: "0.1"
    annotations:
      tekton.dev/pipelines.minVersion: "0.12.1"   
      tekton.dev/tags:token       (just an example, more can be added)
      tekton.dev/displayName: An Optional name for the task 

  spec:
     description: >-
       One line summary of the task. 

       Description of the task 
  ```
  
  see: [#389](https://github.com/tektoncd/catalog/pull/389) | and [Tekton Hub Preview](https://ui-tekton-hub.apps.openshift-web.p0s5.p1.openshiftapps.com/)